app/helpers/application_helper.rb

```diff
@@ -1,2 +1,30 @@
+# Application Helper module
 module ApplicationHelper
+  def bootstrap_class_for(flash_type)
+    { success: 'alert-success',
+      error: 'alert-danger',
+      alert: 'alert-warning',
+      notice: 'alert-info' } [flash_type] || flash_type.to_s
+  end
+
+  def flash_messages
+    flash.each do |msg_type, message|
+      concat(content(msg_type, message))
+    end
+    nil
```

Comment:
Because <code>flash</code> is an instance of <code>ActionDispatch::Flash::FlashHash</code>, and if I don't return nil, it returns an empty hash, and we don't want {} on our screen.